CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Braintree iOS SDK Release Notes
 
+## unreleased
+* BraintreePayPalNativeCheckout (BETA)
+  * Update Package.swift to use binary dependency of `PayPalCheckout` and pulling in the `PayPalCheckout.xcframework.zip` from the `PayPalCheckout` release directly
```

Comment:
Take it or leaving wording nitpick - 
```suggestion
  * Update Package.swift to fetch `PayPalCheckout` binary dependency directly instead of hosting copy in `braintree_ios` repo
```